ma/maSize.cc

```diff
@@ -429,8 +429,11 @@ struct LogAnisoSizeField : public MetricSizeField
     logMEval(f)
   {
     mesh = m;
+    // ADDED
```

Comment:
@avinmoharana 

revert this file back to what it is on develop branch by using

git checkout develop -- ma/maSize.cc